src/guides/v2.3/architecture/tech-stack.md

```diff
@@ -63,8 +63,8 @@ Magento 2.2+ does not support HipHop Virtual Machine (HHVM).
 
 Magento also provides automated testing suites that include unit, integration, functional and performance test scripts, as well as JavaScript tests and tools for static code analysis. Components include PHPUnit for the unit test framework and Selenium for the functional test framework.
 
-This framework is located in the `dev/tests` directory. The functional testing framework `mtf` can be found in a [separate repository](https://github.com/magento/mtf){:target="_blank"}.
-For more information, see the [Functional Testing Framework]({{page.baseurl}}/mtf/mtf_introduction.html) guide.
+This framework is located in the `dev/tests` directory. The functional testing framework `mftf` can be found in a [separate repository](https://github.com/magento/mftf){:target="_blank"}.
```

Comment:
I'm not sure that this is accurate according to current architecture. Maybe @okolesnyk can help with this.

For your information, the magento2 code still contains old `mtf` at [dev/tests/functional](https://github.com/magento/magento2/tree/2.4-develop/dev/tests/functional).

Reply:
Good point. According to our docs, MFTF tests are located in two different places:

- `<magento_root>/app/code/<vendor_name>/<module_name>/Test/Mftf/` is the location of local, customized tests.
- `<magento_root>/vendor/<vendor_name>/<module_name>/Test/Mftf/' is location of tests provided by Magento and vendors.

I believe there is a task to remove MTF test from Magento code that @okolesnyk is working on.

Reply:
Fixed